Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -6395,7 +6397,7 @@ def return_error(message, error='', outputs=None):
             'Type': entryTypes['error'],
             'ContentsFormat': formats['text'],
             'Contents': message,
-            'EntryContext': outputs
+            'EntryContext': outputs,
```

Comment:
🤔 ?

Reply:
[PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)